From 09197b0e7ac90d0abbb7f74be10939a1c2fb89db Mon Sep 17 00:00:00 2001
From: Bruno Haible <bruno@clisp.org>
Date: Fri, 19 Jul 2019 23:23:06 +0200
Subject: [PATCH 3/9] build: Avoid warnings from bison versions >= 3.3.

Reported by Bernhard Voelker <mail@bernhard-voelker.de>.

* gettext-runtime/intl/Makefile.am (YACC): Remove option '-y'.
---
 gettext-runtime/intl/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gettext-runtime/intl/Makefile.am b/gettext-runtime/intl/Makefile.am
index 9f6e576f1..b6d7a1f53 100644
--- a/gettext-runtime/intl/Makefile.am
+++ b/gettext-runtime/intl/Makefile.am
@@ -60,7 +60,7 @@ AM_CPPFLAGS += \
   -Drelocate2=libintl_relocate2 \
   -DDEPENDS_ON_LIBICONV=1
 
-YACC = @INTLBISON@ -y -d
+YACC = @INTLBISON@ -d
 BISONFLAGS = --name-prefix=__gettext
 
 # Tell the ELF linker which symbols to export.
